plugins/permissions/Cargo.toml

```diff
@@ -9,7 +9,10 @@ license = "Apache-2.0"
 keywords = ["permissions", "android", "ios"]
 readme = "README.md"
 
-[dependencies]
-
 [target.'cfg(target_os = "android")'.dependencies]
+ndk-glue = "0.6.0"
+
+[dependencies]
 jni = "0.19"
+ndk-context = "0.1"
```

Comment:
All ndk/jni should be in target_os android deps. Move it to ndk-glue.

Reply:
Will return as was it after my refactoring done